secret_number.js

```diff
@@ -1,4 +1,9 @@
+/*jshint esversion: 6 */
 'use strict';
 module.exports = function() {
-  
-};
\ No newline at end of file
+    let secretNumber = (Math.random()*1000001);
```

Comment:
So (Math.random()*1000001) at it's largest can be 1000000.99999. This is slightly larger than the max value of 1000000. Can you think of a Math function that allows the max value to be exactly 1000000.

Also while let is fine, you could use const here and it would be more appropriate.